migration/cuju-kvm-share-mem.c

```diff
@@ -1098,8 +1129,17 @@ static void* trans_ram_conn_thread_func(void *opaque)
         s->ram_len += ret;
 
         ret = kvm_start_kernel_transfer(s->cur_off, s->ram_fds[d->index], d->index, ft_ram_conn_count);
-
-        assert(ret >= 0);
+        if(ret<0)
+        {
+            //printf("ret<0    %d!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",ret);
```

Comment:
Please refine this description

Reply:
fix it to printf("%s ret<0 \n", __func__);